src/Ledger/PParams.lagda

```diff
@@ -47,40 +47,59 @@ record PParams : Set where
 \emph{Network group}
 \AgdaTarget{maxBlockSize, maxTxSize, maxHeaderSize, maxValSize, pv}
 \begin{code}
-        maxBlockSize maxTxSize maxHeaderSize maxValSize  : ℕ
-        pv                                               : ProtVer -- retired, keep for now
+        maxBlockSize       : ℕ
+        maxTxSize          : ℕ
+        maxHeaderSize      : ℕ
+        maxValSize         : ℕ
```

Comment:
Why did you revert this completely? I believe the best presentation here is your initial change without the vertical alignment.

Reply:
@WhatisRT WDYT?